contracts/Kiftables.sol

```diff
@@ -325,12 +346,14 @@ contract KiftVans is ERC721, IERC2981, Ownable, ReentrancyGuard, VRFConsumerBase
         public
         view
         virtual
-        override
+        override(ERC721A)
         returns (string memory)
     {
-        require(_exists(_tokenId), "Nonexistent token");
+        require(_exists(_tokenId), "Nonexistent token"); // does this need to be here?
 
-        if (id >= CONTRIBUTOR_OFFSET + lastTokenRevealed) {
+        // this cant be >= otherwise the last token cant be revealed
+        // @Lev, can you confirm this is without >= ?
+        if (_tokenId > lastTokenRevealed) {
```

Comment:
yea i guess the >= was when tokenId started at 0. but theres still the question of lastTokenRevealed starting at 0 but startTokenId starting at 1

Reply:
if we set them both to 0, then using >= will incorrectly map token 0 to 0.json, I think they need to be offset by 1 otherwise the newest minted token will be revealed at 0, 1000, 2000 etc